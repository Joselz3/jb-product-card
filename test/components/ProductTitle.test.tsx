import React from 'react'
import {ProductTitle} from '../../src/components'
import renderer from 'react-test-renderer'


describe('ProductTitile',()=>{
    test('debe mostrar el comoponente correctamente',()=>
    {
      const wrapper = renderer.create(
        <ProductTitle title='costum product'></ProductTitle>
      )

      expect(wrapper.toJSON()).toMatchSnapshot()

    })

    test('debe de mostrar el componente con el nombre del producto',()=>{
      
    })
})