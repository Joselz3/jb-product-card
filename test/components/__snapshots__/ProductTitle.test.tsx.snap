// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProductTitile debe mostrar el comoponente correctamente 1`] = `
<span
  className="productDescription undefined"
>
  costum product
</span>
`;
